ambry-router/src/main/java/com/github/ambry/router/NonBlockingRouter.java

```diff
@@ -78,12 +82,7 @@
   private final Time time;
```

Comment:
we probably don't need most of those variables in nonblobkingrouter anymore.

Reply:
+1. Also, there are few unused imports.

Reply:
fixed.